src/span_context.js

```diff
@@ -124,8 +155,13 @@ export default class SpanContext {
     return this._samplingFinalized;
   }
 
-  set traceId(traceId: Buffer): void {
-    this._traceId = traceId;
+  set traceIdLow(traceIdLow: Buffer): void {
```

Comment:
why do we need to expose low/high fields explicitly?  It seems like the Buffer type can represent the full ID as a single buffer, and do it transparently to the user.

Reply:
ok, I'll work on that